modules/Blockchain/OriginTrailParachain/migrations/2_total_migration.js

```diff
@@ -0,0 +1,228 @@
+            LitigationStorage,
+            hub.address,
+        );
+        await hub.setContractAddress('LitigationStorage', litigationStorage.address);
+
+        marketplaceStorage = await deployer.deploy(
+            MarketplaceStorage,
+            hub.address,
+            { gas: 6000000, from: accounts[0] },
+        );
+        await hub.setContractAddress('MarketplaceStorage', marketplaceStorage.address);
+
+        profile = await deployer.deploy(Profile, hub.address);
+        await hub.setContractAddress('Profile', profile.address);
+
+        holding = await deployer.deploy(Holding, hub.address);
+        await hub.setContractAddress('Holding', holding.address);
+
+        litigation = await deployer.deploy(
+            Litigation,
+            hub.address,
+        );
+        await hub.setContractAddress('Litigation', litigation.address);
+
+        marketplace = await deployer.deploy(
+            Marketplace,
+            hub.address,
+            { gas: 7000000, from: accounts[0] },
+        );
+        await hub.setContractAddress('Marketplace', marketplace.address);
+
+        replacement = await deployer.deploy(
+            Replacement,
+            hub.address,
+        );
+        await hub.setContractAddress('Replacement', replacement.address);
+
+        console.log('\n\n \t Contract adressess on starfleet:');
+        console.log(`\t Hub contract address: \t\t\t${hub.address}`);
+        console.log(`\t Profile contract address: \t\t${profile.address}`);
+        console.log(`\t Holding contract address: \t\t${holding.address}`);
+        console.log(`\t Litigation contract address: \t\t${litigation.address}`);
+        console.log(`\t Marketplace contract address: \t\t${marketplace.address}`);
+        console.log(`\t Replacement contract address: \t\t${replacement.address}`);
+
+        console.log(`\t ProfileStorage contract address: \t${profileStorage.address}`);
+        console.log(`\t HoldingStorage contract address: \t${holdingStorage.address}`);
+        console.log(`\t LitigationStorage contract address: \t${litigationStorage.address}`);
+        console.log(`\t MarketplaceStorage contract address: \t${marketplaceStorage.address}`);
+
+        break;
+    case 'updateContracts':
+        console.log(`Deploying from wallet: ${accounts[0]}`);
+        await Hub.at('0x43C7faf221DFd7f10a07F646D5635766021D036E')
+            .then((result) => {
+                hub = result;
+            });
+
+        profile = await deployer.deploy(Profile, hub.address);
+        await hub.setContractAddress('Profile', profile.address);
+
+        console.log('\n\n \t Contract adressess on starfleet:');
+        console.log(`\t Hub contract address: \t\t\t${hub.address}`);
+        console.log(`\t Profile contract address: \t\t${profile.address}`);
+
+        break;
+    default:
+        console.warn('Please use one of the following network identifiers: ganache, mock, test, or rinkeby');
```

Comment:
I think this is outdated, we have development, ganache, test, testnet, contracts, updateContracts